packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -18,7 +18,7 @@ We use Red Hat Display and Red Hat Text.
 <Button style={{borderRadius: '0px', fontWeight: '600', paddingTop: '12px', paddingBottom: '12px', paddingLeft: '24px', paddingRight: '24px'}} variant="primary" component="a" href="https://github.com/RedHatOfficial/RedHatFont" target="_blank">DOWNLOAD</Button>
 
 ## Usage guidelines
-Use our typographic styles to communicate visual hierarchy. A consistent and logical hierarchy provides a clear pattern for users, making it easier to quickly scan and understand information on a page.
+Use typography to create visual hierarchy. A consistent and logical hierarchy makes it easier for users to quickly scan and understand information on a page.
 
 <TitleLevel
   asGrid
```

Comment:
Line 26: What does "login" mean on line 26? Are we referring to a login screen or field? Also, let's remove the "etc." and instead go with something along the lines of: *Not to be used in content block (such as landing pages and login screens)  

Line 34: Remove the "etc." so that we have: *Some components use RedHatText at 700 font weight, which is RedHatText Medium (such as alerts and navigation) >> instead of "such as," we can also do "example," or "for example,"

Line 40: To keep these notes consistent, let's remove the period and capitalization of the first word in the parentheses. So we would have something like: *Not to be used in content block (only used with data visualizations when 14px is not small enough)

Reply:
Thanks for catching these!

Reply:
Np! I reviewed your changes, and they look great, but the pesky period is still in there after "enough" in "*Not to be used in content block (only used with data visualizations when 14px is not small enough.)" Once you remove that, we should be good to go!

Reply:
Oops! Just fixed.